apps/ledger-live-mobile/src/components/Web3AppWebview/WalletAPIWebview.tsx

```diff
@@ -38,7 +43,16 @@ export const WalletAPIWebview = forwardRef<WebviewAPI, WebviewProps>(
         renderError={() => <NetworkError handleTryAgain={() => webviewRef.current?.reload()} />}
         testID="wallet-api-webview"
         allowsUnsecureHttps={__DEV__ && !!Config.IGNORE_CERTIFICATE_ERRORS}
-        {...webviewProps}
+        {...webviewStateProps}
+        onLoadEnd={event => {
+          if (onLoadEnd) {
+            onLoadEnd(event);
+
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore next-line
+            this.isLoading = event.nativeEvent.loading;
```

Comment:
TS is correct here IMO
You don't have a `this` in this context
The exemple present in the documentation you shared is probably just showing a use case in a class component
Also there is no `isLoading` in the codebase of `react-native-webview` so I don't think this code does anything

Reply:
I thought there would be more to the underlying component as I checked and also couldn't see the isLoading declared anywhere.  `this` is available but `isLoading` is undefined so can see that it will make no actual difference.  When I was initially testing the differences I could reproduce the infinite loading only when this line was not being set, but now it is not being reproduced even when it is not.

I suspect it may be a memory issue or race condition for when `onLoadEnd` and `onNavigationStateChange` run.